app/src/main/java/de/westnordost/streetcomplete/quests/seating/Seating.kt

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.quests.seating
+
+enum class Seating {
+    NO,
+    ONLY_INDOOR,
+    ONLY_OUTDOOR,
+    INDOOR_AND_OUTDOOR,
+}
```

Comment:
You can give some extra attributes to enum members, which can simplify the logic in `applyAnswerTo` (see my next comment):

```suggestion
enum class Seating(val hasOutdoorSeating: Boolean, val hasIndoorSeating: Boolean) {
    NO(false, false),
    ONLY_INDOOR(false, true),
    ONLY_OUTDOOR(true, false),
    INDOOR_AND_OUTDOOR(true, true),
}
```